scripts/maint/release.py

```diff
@@ -132,12 +132,36 @@ def main(args):
     # While we use Github releases and not dist.tpo
     print("\n6. Create release tarball")
     print("-------------------------")
+
+    print("\nCreating a release tarball...")
+    subprocess.call(
+        "git archive --format=tar.gz --prefix=sbws-{}/ "
+        "-o v{}.tar.gz v{}"
+        .format(release_version, release_version, release_version).split(' ')
+    )
+    print("\nCreating tarball hash file...")
+    fd = open('v{}.tar.gz.sha256'.format(release_version), 'w')
+    subprocess.call("sha256sum v{}.tar.gz".format(release_version).split(' '),
+                    stdout=fd)
+    fd.close()
+
     print("Obtaining Github tarball...")
+    # This will overwrite local tarball, but that's fine since the hash file
```

Comment:
Overwriting the local tarball is only fine if the user reads the output of the script, and notices when the GitHub tarball is different. Please put the tarballs in separate files.